crates/rethnet/src/lib.rs

```diff
@@ -97,57 +152,79 @@ where
     let args = Cli::parse_from(args);
     match args.command {
         Command::Node(node_args) => {
-            tracing_subscriber::fmt::Subscriber::builder()
-                .with_max_level(match node_args.verbose {
-                    0 => Level::ERROR,
-                    1 => Level::WARN,
-                    2 => Level::INFO,
-                    3 => Level::DEBUG,
-                    4 => Level::TRACE,
-                    _ => Err(anyhow!(
-                        "Specifying --verbose more than 4 times is unsupported"
-                    ))?,
-                })
-                .init();
-
-            let server = rethnet_rpc_server::Server::new(
-                server_config_from_cli_args_and_config_file(node_args, ConfigFile::default())?,
-            )
-            .await?;
-
-            async fn await_signal() {
-                use tokio::signal;
-
-                let ctrl_c = async {
-                    signal::ctrl_c()
-                        .await
-                        .expect("failed to install Ctrl+C handler");
-                };
-
-                #[cfg(unix)]
-                let terminate = async {
-                    use signal::unix::{signal, SignalKind};
-                    signal(SignalKind::terminate())
-                        .expect("failed to install signal handler")
-                        .recv()
-                        .await;
+            if node_args.init_config_file {
```

Comment:
Thank you for asking, I hadn't considered it. My intention was that when `--init-config-file` is specified, the binary does nothing else besides generate that default config file to `edr.toml`. I _think_ that having the help message specify that it specifically writes to `edr.toml` precludes any expectation that you could specify something otherwise, via `--config-file`, but I'm open to suggestions on improving that perception.

Reply:
If you just want to it to generate a file, I would create a separate command name `Command::Init` and use that for it.

That seems easier than untangling the different options between a shared `--init-config-file` and `--config-file`

Reply:
Great idea, thanks, done in https://github.com/NomicFoundation/hardhat/pull/4221/commits/953c0895a5793c6ea685372036dcb5097ffa0aed